manifests/katello.pp

```diff
@@ -55,7 +55,7 @@
     # (especially in the custom certs scenario)
     files            => ["${katello_rhsm_setup_script_location}:755=${katello_www_pub_dir}/${katello_rhsm_setup_script}"],
     bootstrap_script => "/bin/bash ${katello_rhsm_setup_script_location}",
-    postun_script    => 'test -f /etc/rhsm/rhsm.conf.kat-backup && command cp /etc/rhsm/rhsm.conf.kat-backup /etc/rhsm/rhsm.conf',
+    postun_script    => template('certs/postun.erb'),
```

Comment:
If there's no template I'd prefer this to be a static file. You'll need to move it to `files/postun.sh` as well.
```suggestion
    postun_script    => file('certs/postun.sh'),
```

Reply:
Learned something new!  Done!